### PR TITLE
CLOUD-343 Operatror version upgrade

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/Percona-Lab/percona-helm-charts?pullRequest=4) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/Percona-Lab/percona-helm-charts?pullRequest=4) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/Percona-Lab/percona-helm-charts?pullRequest=4) it.</sub>

File: charts/pxc-database/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for installing Percona XtraDB Cluster Databases using the PXC Operator.
-name: pxc-database
+name: pxc-db
 home: https://www.percona.com/doc/kubernetes-operator-for-pxc/kubernetes.html
 version: 0.1.4
```
Comment (paulczar) [inline charts/pxc-database/Chart.yaml]: always bump the chart version on a change using semver scheme

File: charts/pxc-database/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 appVersion: "1.0"
```
Comment (paulczar) [inline charts/pxc-database/Chart.yaml]: appVersion should bump to 1.1.0 to match the operator version.

File: charts/pxc-database/templates/_helpers.tpl [hunk 3]
```diff
@@ -13,13 +13,13 @@ If release name contains chart name it will be used as a full name.
 */}}
 {{- define "pxc-database.fullname" -}}
 {{- if .Values.fullnameOverride -}}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- .Values.fullnameOverride | trunc 21 | trimSuffix "-" -}}
```
Comment (paulczar) [inline charts/pxc-database/templates/_helpers.tpl]: we truncate at 63 because of kube limitations on some names ( dns etc ). these are defaults created by `helm create` chart generator.   why the change to 21 ?

(same question for the trunc 3 etc )

File: charts/pxc-database/Chart.yaml (hunk 1 above)
Comment (cap1984) [reply charts/pxc-database/Chart.yaml]: ok

File: charts/pxc-database/Chart.yaml (hunk 2 above)
Comment (cap1984) [reply charts/pxc-database/Chart.yaml]: ok

File: charts/pxc-database/templates/_helpers.tpl (hunk 3 above)
Comment (cap1984) [reply charts/pxc-database/templates/_helpers.tpl]: `status:
  conditions:
  - lastTransitionTime: "2019-07-25T08:55:04Z"
    message: 'wrong PXC options: cluster (pxc) name too long, must be no more than
      22 characters'
    reason: ErrorReconcile
    status: "True"
    type: Error
  message:
  - 'Error: wrong PXC options: cluster (pxc) name too long, must be no more than 22`

Comment (cap1984) [reply charts/pxc-database/templates/_helpers.tpl]: discrepancy between trim length is just a typo

File: charts/pxc-db/templates/_helpers.tpl [hunk 4]
```diff
@@ -13,13 +13,13 @@ If release name contains chart name it will be used as a full name.
...
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- .Values.fullnameOverride | trunc 21 | trimSuffix "-" -}}
 {{- else -}}
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- if contains $name .Release.Name -}}
-{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- .Release.Name | trunc 21 | trimSuffix "-" -}}
 {{- else -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 21 | trimSuffix "-" -}}
```
Comment (delgod) [inline charts/pxc-db/templates/_helpers.tpl]: should we also trunc line 31?

File: charts/pxc-database/templates/_helpers.tpl (hunk 3 above)
Comment (paulczar) [reply charts/pxc-database/templates/_helpers.tpl]: ahhha!!

File: charts/pxc-db/templates/_helpers.tpl (hunk 4 above)
Comment (cap1984) [reply charts/pxc-db/templates/_helpers.tpl]: done
